### DISCUSSION TITLE
What ideas do you have for a Civic Accessibility Summit?

### DISCUSSION THREAD

**Discussion by lukefretwell** [CivicActions/accessibility#264] [Ideas]
We are organizing a Civic Accessibility Summit. 

Questions:

* Who should we invite to speak?
* What topics should we address?
* What technologies should we employ to make it as accessible as possible?
* What other ideas/thoughts do you have?

**Comment by mgifford** [comment]
I think it would be interesting to talk about the authoring interface.  
I can think of a lot of folks I'd be interested in inviting, but some of them would need compensation. 
Who would we like to participate? How would we like to involve our team, our clients, and the accessibility community? 

axe-con was a great example of an accessible conference.
- Zoom has lots of great accessibility features built-in.  They wanted to scale this up big (and it was bigger than what they expected)
- They piped it through another live stream software (vMix) to make it as accessible as it could. vMix itself is not considered accessible, it was used more as a means to an end. From vMix, the live steam was sent to Vimeo, which acted as the primary user interface.
- Now build an accessible webpage container within Vimeo.  
- Slido was used for chat
- Captioning was difficult - you can't have both speed & accuracy. They opted for accuracy which required a 21 sec delay.
- Wordpress pulled all of these elements together for a consistent user interface.

Given we would be doing something much smaller than axe-con, I think Zoom will be sufficient.
